cosmos/grand.json

```diff
@@ -0,0 +1,61 @@
+{
+  "chainId": "grand-1",
+  "chainName": "Noble",
+  "rpc": "https://rpc.testnet.noble.strange.love",
+  "rest": "https://api.testnet.noble.strange.love/",
+  "nodeProvider": {
+    "name": "Strangelove",
+    "email": "support@strange.love",
+    "website": "https://strange.love/"
+  },
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/noble/chain.png",
```

Comment:
@boojamya 
The chain image file should go in the folder of that chainIdentifier!
So, Could you make a directory `images/grand` and add `chain.png` image file.
after that Could you change `chainSymbolImageUrl` field?

Reply:
Done! 
Just pointing out the obvious here, but grand-1 is the testnet version of noble-1 so they are nearly identical. 

lmk if there is anything else lol.